src/pulp_docs/cli.py

```diff
@@ -43,7 +48,57 @@ def draft_callback(ctx: click.Context, param: click.Parameter, value: bool) -> b
     help="Don't fail if repositories are missing.",
 )
 
+
+async def clone_repositories(repositories: List[str], dest_dir: Path) -> None:
```

Comment:
The python version we use allows this easier annotation:
```suggestion
async def clone_repositories(repositories: list[str], dest_dir: Path) -> None:
```

(Also in my experience `import typing as t` has been way more convenient than `from typing import ...`.)